addons/skin.estuary/xml/VideoFullScreen.xml

```diff
@@ -34,6 +34,8 @@
 		<control type="group" id="1">
 			<depth>DepthOSD+</depth>
 			<visible>Player.Caching</visible>
+			<animation effect="fade" time="0">Visible</animation>
+			<animation effect="fade" time="0">Hidden</animation>
```

Comment:
these animations don't have any effect i think?

Reply:
yeah, they can go... a fade animation with time set to 0 doesn't do anything.

Reply:
Thanks @ronie, I  will remove the two anims.